fem/complex_fem.hpp

```diff
@@ -79,11 +117,40 @@ class ComplexGridFunction : public Vector
    /// @a gfr and @a gfi to match the ComplexGridFunction.
    void SyncAlias() { gfr->SyncAliasMemory(*this); gfi->SyncAliasMemory(*this); }
 
+   virtual real_t ComputeL2Error(Coefficient &exsolr, Coefficient &exsoli,
+                                 const IntegrationRule *irs[] = NULL) const
+   {
+      real_t err_r = gfr->ComputeL2Error(exsolr, irs);
+      real_t err_i = gfi->ComputeL2Error(exsoli, irs);
+      return sqrt(err_r * err_r + err_i * err_i);
```

Comment:
```suggestion
      return hypot(err_r, err_i);
```
A matter of taste, I guess, but `hypot()` is a bit safer also.